src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/APIPermissionChecker.kt

```diff
@@ -30,6 +30,10 @@ class APIPermissionChecker(url: String,
             {
                 requestHelper.postWithPermissions(this.url, this.postData, this.contentType, permissions)
             }
+            HttpMethod.delete ->
+            {
+                requestHelper.deleteWithPermissions(this.url, this.contentType, permissions)
```

Comment:
Want to update the else clause below so that the error message is `Only GET, POST and DELETE supported`?